pages/developers/gha_pure.md

```diff
@@ -128,31 +128,70 @@ later in the upload action for the release job, as well).
 
 And then, you need a release job:
 
+<div class="skhep-bar d-flex m-2" style="justify-content:center;">
+  <button class="skhep-bar-item oidc-btn btn m-2 btn-purple" onclick="openTab('oidc')">Trusted Publishing</button>
+  <button class="skhep-bar-item token-btn btn m-2" onclick="openTab('token')" id='token-btn'>Token</button>
+</div>
+
+<div class="skhep-tab oidc-tab" markdown="1">
+
 {% raw %}
 
 ```yaml
-publish:
+upload_all:
   needs: [dist]
+  environment: pypi
+  permissions:
+    id-token: write
   runs-on: ubuntu-latest
   if: github.event_name == 'release' && github.event.action == 'published'
+  steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: artifact
+        path: dist
+
+    - uses: pypa/gh-action-pypi-publish@release/v1
+```
 
+{% endraw %}
+
+When you make a GitHub release in the web UI, we publish to PyPI. You'll just
+need to tell PyPI which org, repo, workflow, and set the `pypi` environment to
+allow pushes from from GitHub. If it's the first time you've published a
+package, go to
+https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/ for
```

Comment:
Can actually link to it with some human readable text?

```suggestion
package, go to the [PyPI trusted publisher docs] for
```

and then at the bottom of the page with links

```markdown
[PyPI trusted publisher docs]: https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/
```